onlyfans-dl.py

```diff
@@ -399,6 +440,8 @@ def get_all_photos(images):
 
         download_public_files()
 
+        messages = api_request("/chats/" + PROFILE_ID + "/messages", getdata={"limit": str(POST_LIMIT), "order": "asc"})
```

Comment:
Not working for me, only downloading 1 message. I had to add `"offset": str(1)` so it could work properly